### ISSUE TITLE
Ubuntu Touch detection

### ISSUE THREAD

**Issue by VMbuseck** [ded/bowser#94]
I am missing a correct detection of the Ubuntu Touch smartphone device.

"Mozilla/5.0 (Linux; Ubuntu 14.04 like Android 4.4) AppleWebKit/537.36 Chromium/35.0.1870.2 Mobile Safari/537.36"

**Comment by lancedikson** [comment]
@VMbuseck, what the browser is this?

**Comment by VMbuseck** [comment]
This is the inbuilt browser of the Ubuntu Touch Phone OS: https://launchpad.net/webbrowser-app

**Comment by lancedikson** [comment]
@VMbuseck, ok, we will add it if you help me to find pretty name of the browser. All that I found is the `webbrowser-app` [source](http://askubuntu.com/questions/307912/what-browser-does-ubuntu-touch-use). Is this the real name of the browser? Or there is some more readable name?

**Comment by VMbuseck** [comment]
The official name seems to be "webbrowser-app". So "Ubuntu Touch Webbrowser" would be a pretty name.

**Comment by lancedikson** [comment]
@VMbuseck, ok. I've analysed all aspects of the question and decided that we can't say that it's `Ubuntu Touch Webbrowser` using this UA. I mean, I think there are some devices which are on Linux "like Android" and uses Chromium as basic browser. So, I've added support of Chromium and now you will get detection of Chromium-browser on mobile linux:

```
        mobile: true
      , linux: true
      , chromium: true
      , version: '35.0'
      , blink: true
      , x: true
```

Using this data you can decide in your app that you have got "Ubuntu Touch Webbrowser". :) 
Is it ok for you?

**Comment by lancedikson** [comment]
released in 1.3.0
